modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/test/java/com/liferay/dynamic/data/mapping/internal/io/exporter/DDMFormInstanceRecordWriterTrackerImplTest.java

```diff
@@ -35,35 +43,48 @@ public class DDMFormInstanceRecordWriterTrackerImplTest {
 	public static final LiferayUnitTestRule liferayUnitTestRule =
 		LiferayUnitTestRule.INSTANCE;
 
+	@Before
+	public void setUp() {
+		_ddmFormInstanceRecordWriterTrackerImpl =
+			new DDMFormInstanceRecordWriterTrackerImpl();
+
+		_bundleContext = SystemBundleUtil.getBundleContext();
```

Comment:
@JanisZhang Please double check where `_bundleContext` is usually initialized, in `setUp` or `setUpClass`, and follow the common practice.